projects/mercury/src/metadata/views/facets/DateSelectionFacet.js

```diff
@@ -33,33 +33,23 @@ const DateSelectionFacet = (props: MetadataViewFacetProperties) => {
         }
     }, [activeFilterValues]);
 
-    const isValid = (val: Date | null): boolean => {
```

Comment:
I'm not sure if fully removing this check is a good solution, now I can type anything, like 99/99/9999 and it doesn't show it's invalid date (not in red)...

Reply:
ah that's not good indeed, that needs some rework